cmd/kubectl-testkube/commands/connect.go

```diff
@@ -0,0 +1,163 @@
+package commands
+
+import (
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/pterm/pterm"
+	"github.com/skratchdot/open-golang/open"
+
+	cloudclient "github.com/kubeshop/testkube/pkg/cloud/client"
+	"github.com/kubeshop/testkube/pkg/ui"
+	"github.com/spf13/cobra"
+)
+
+func NewInteractiveCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "connect",
+		Aliases: []string{"c"},
+		Short:   "Testkube Cloud interactive mode",
+		Long:    `Interactive mode for Testkube Cloud migrations`,
+		Run:     cloudConnect,
+	}
+
+	return cmd
+}
+
+const (
+	listenAddr      = "127.0.0.1:7899"
+	redirectUrl     = "http://" + listenAddr
+	tokenQueryParam = "token"
+)
+
+func cloudConnect(cmd *cobra.Command, args []string) {
+
+	header := pterm.DefaultHeader.WithBackgroundStyle(pterm.NewStyle(pterm.BgDefault, pterm.FgLightMagenta, pterm.Bold)).WithMargin(0)
+	text := pterm.DefaultHeader.WithBackgroundStyle(pterm.NewStyle(pterm.BgDefault, pterm.FgDarkGray)).
+		WithTextStyle(pterm.NewStyle(pterm.BgDefault, pterm.FgGray)).WithMargin(0)
+
+	header.Println("Connect your cloud environment:")
+	text.Println("You can learn more about connecting your Testkube instance to the Cloud here:\nhttps://docs.testkube.io/etc")
+	header.Println("You can safely switch between connecting Cloud and disconnecting without losing your data.")
+
+	// TODO: implement context info
+	header.Println("Current status of your Testkube instance")
+	text.Println(`
+Context:    On-Premise (local OSS context)
+Cluster:    cluster
+Namespace:  testkube
+`)
+
+	header.Println("Login")
+	text.Println("Please open the following link in your browser and log in:\nhttps://cloud.testkube.io/login?redirect_uri=....")
+
+	// open browser with login page and redirect to localhost:7899
+	open.Run("https://cloud.testkube.io/?redirectUri=" + url.QueryEscape(redirectUrl))
```

Comment:
might be move it to const?

Reply:
aaa yeah - still working on it - it should be draft